packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -65,6 +65,13 @@ Class name added to the placeholder.
 - Type: `String`
 - Required: No
 
+### icon
+
+Dashicon to display left of the title.
+
+- Type: `String`
```

Comment:
The icon can also be a normal component instead of a string. I think we should add `|WPComponent` to the type.

Reply:
Yes, you're right.  I didn't notice that until recently.  Thanks!